assembly/src/parsers/u32_ops.rs

```diff
@@ -1,5 +1,13 @@
 use super::{parse_int_param, push_value, AssemblyError, Felt, Operation, Token, Vec};
 
+#[derive(PartialEq)]
+pub enum Type {
+    Checked,
+    Unchecked,
+    Wrapping,
+    Overflowing,
+}
```

Comment:
I would not call this `Type` as it is way too generic. Maybe something like `U32OpType` or `U32OpCategory`.